src/librustc_trans/lib.rs

```diff
@@ -37,6 +37,11 @@
 use rustc::dep_graph::WorkProduct;
 use syntax_pos::symbol::Symbol;
 
+#[allow(unused_extern_crates)]
```

Comment:
Can you add a comment here pointing to the `Cargo.toml` of rustc_cratesio_shim?

Reply:
Done.